pkg/cri/opts/spec_linux.go

```diff
@@ -446,6 +446,13 @@ func WithResources(resources *runtime.LinuxContainerResources, tolerateMissingHu
 				logrus.Warn("hugetlb cgroup controller is absent. skipping huge pages limits")
 			}
 		}
+
+		if resources.Unified != nil {
```

Comment:
let's use the get.. and check if s.Linux.Resources.Unified is nil 

```
if unified := resources.GetUnified(); unified != nil {
    if s.Linux.Resources.Unified == nil {
        s.Linux.Resources.Unified = make(map[string]string)
    }
    for k, v := range unified {
         s.Linux.Resources.Unified[k] = v
    }
}

```

Reply:
Sorry, fix it.